drivers/flash/soc_flash_nrf.c

```diff
@@ -352,6 +359,44 @@ static void time_slot_callback_helper(u32_t ticks_at_expire, u32_t remainder,
 	}
 }
 
+static void time_slot_callback_prepare(u32_t ticks_at_expire, u32_t remainder,
+				       u16_t lazy, void *context)
+{
+	u8_t instance_index;
+	u8_t ticker_id;
+	int err;
+
+	ll_timeslice_ticker_id_get(&instance_index, &ticker_id);
```

Comment:
The time_lost_callback_prepare and time_slot_callback_abort are quite similar to each other, from that line to the end of function, with exception to callback and ticks provided to ticker_start. Maybe it would be possible to dump similar code into helper function?

Reply:
I will have a helper function and push updates

Reply:
Thanks.